### PR TITLE
fix slug event type no repeat

### PR DESCRIPTION
**Pull request by alejandroaguilarhiguera**

### CODE REVIEW

File: lib/schemas/eventType.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+          ),
+      }),
+  }),
+
+  delete: Joi.object({
+    id: Joi.number().required(),
+  }),
+};
+
+export default eventTypeSchema;
```
Comment (zomars) [inline lib/schemas/eventType.ts:122]: This is a nice schema, but... We already use `zod` as a schema validator. It wouldn't make sense to have two validation libraries.

https://www.npmjs.com/package/zod

Comment (alejandroaguilarhiguera) [reply lib/schemas/eventType.ts:117]: it's fixed

**Comment by PeerRich** [comment]
@alejandroaguilarhiguera can you explain more what this PR is doing?

**Comment by alejandroaguilarhiguera** [comment]
> @alejandroaguilarhiguera can you explain more what this PR is doing?

I can mistakenly edit the event type slug to another with the same slug, the pull request validates that I cannot repeat it

Captura de Pantalla 2021-12-27 a la(s) 10 17 14 a m

another event type
Captura de Pantalla 2021-12-27 a la(s) 10 22 51 a m

Result: 404 not found on preview

**Comment by zomars** [comment]
Sorry @alejandroaguilarhiguera but this PR is handling too much in my opinion. We should open up an issue with if this bug is still happening and handle it in a simpler way, shall we? I'm going to close this for now.

**Pull request closed without merging**